roles/pulp/tasks/install.yml

```diff
@@ -127,3 +132,4 @@
 
   become: true
   become_user: '{{ pulp_user }}'
+...
```

Comment:
Just curious why this was necessary

Reply:
https://yaml.org/spec/1.2/spec.html Section 2.2
No, it is not. But the `---` isn't either.

Reply:
AFAIK, --- is just so the linter doesn't yell at us.

Reply:
I got used to writing the `...` but i can remove it again. Thought it was considered best practice.

Reply:
Doesn't bother me :)